apis/alarm/src/lib.rs

```diff
@@ -70,6 +70,17 @@ impl<S: Syscalls, C: platform::subscribe::Config> Alarm<S, C> {
             .map(Hz)
     }
 
+    pub fn get_ticks() -> Result<u32, ErrorCode> {
+        S::command(DRIVER_NUM, command::TIME, 0, 0).to_result()
+    }
+
+    pub fn get_milliseconds() -> Result<u64, ErrorCode> {
+        let ticks = Self::get_ticks()? as u64;
+        let freq = (Self::get_frequency()?).0 as u64;
```

Comment:
It's a real bummer that this would induce an additional system call each time, even though the frequency generally won't chance. Seems like this is a good place for a lazily initialized static or something...

Reply:
It's a good use case for the read only syscalls as well. But libtock-rs currently doesn't support them